mastering-plone/views_2.rst

```diff
@@ -403,7 +403,7 @@ We will have to either reinstall our add-on or run the GenericSetup import step
 
 ..  note::
 
-    To change it ttw go to the ZMI (http://localhost:8080/Plone/manage), go to ``portal_types`` and select the type for which the new view should be selectable (*talk*).
+    To change it TTW go to the ZMI (http://localhost:8080/Plone/manage), go to ``portal_types`` and select the type for which the new view should be selectable (*talk*).
```

Comment:
```
:guilabel:`portal_types`
```

Ditto for "talk"

Reply:
Why :guilabel: here but double back ticks above?

Reply:
I assumed it was intended to be an inline literal, but if it is a guilabel, then use that.